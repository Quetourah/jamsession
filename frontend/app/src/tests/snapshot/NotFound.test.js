import React from 'react';
import NotFound from '../../containers/NotFound';
import renderer from 'react-test-renderer';

it('renders login page correctly', () => {
  const tree = renderer
    .create(<NotFound/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});


