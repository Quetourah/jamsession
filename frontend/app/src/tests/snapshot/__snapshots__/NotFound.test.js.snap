// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login page correctly 1`] = `
<div
  className="NotFound"
>
  <h3>
    Sorry, page not found!
  </h3>
</div>
`;
